lib/internal/process/next_tick.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const kMaxCallbacksPerTick = 1e4;
```

Comment:
Heh. Sorry. I wasn't being literal in the name `kMaxCallbacksUntilQueueIsShortened`. For an actual name, possibly `kMaxCallbacksPerLoop`. Though I won't wait until you make the change to sign-off so you won't have to wait for me.